src/utils/dispute-utils.js

```diff
@@ -271,23 +284,24 @@ export function getAdjudicationPhase(dispute, round, date, courtConfig) {
     }
   }
 
-  // If given term is before the reveal start term of the last round, then jurors are still allowed to commit votes for the last round
-  const revealTermStartTime = draftTermEndTime + commitTerms * termDuration
-  if (date.isBefore(revealTermStartTime)) {
+  // Jurors can commit their votes between when the commit phase term starts and ends
+  // Note that the commit start term is the same as the draft end term
```

Comment:
Wondering: if the time has passed, but the term hasn't updated, can a juror still commit/reveal their vote?

Reply:
Yes, see https://github.com/aragon/court-dashboard/pull/335#discussion_r428656118

Reply:
Perhaps we can add this clarification as a comment and as an explanation of why we'd use the current court term in the contracts to calculate whether they can still commit/reveal their vote.

Reply:
Update to this: we've clarified that the above ("if the time has passed, but the term hasn't updated, can a juror still commit/reveal their vote?") will **NOT** work. Commiting/revealing will advance the court term.

We will use the expected court term in the dispute detail view.